pages/events/event_view.php

```diff
@@ -52,15 +53,28 @@
             </div>
         </div>
 
+        <?php if ($has_file): ?>
+            <div class="file-button">
+                <a href="/download?id=<?= $event["circu"] ?>" , role="button" , class="">
```

Comment:
Your HTML looks weird here: you don't need commas and your class attribute is empty

Reply:
Fixed !